pages/ShibbolethHandler.inc.php

```diff
@@ -432,7 +562,6 @@ function _shibbolethRedirect($request) {
 			null,
 			true
 		);
-		$shibUrl = $wayfUrl . '?target=' . $shibLoginUrl;
-		return $request->redirectUrl($shibUrl);
+		return "/" . $wayfUrl . '?target=' . $shibLoginUrl;
```

Comment:
Why force the leading slash here?  Shouldn't it be up to the admin to configure an absolute or relative path?

Reply:
I had an issue with single context journals. For example, in my test single context site it would set the redirect to `https://example.com/journalName/Shibboleth.sso/Login` when it was supposed to be just `https://example.com/Shibboleth.ss/Login`.

Reply:
But if the admin entered "/Shibboleth.sso/Login" rather than "Shibboleth.sso/Login", it would still work, true?

Reply:
Hmm true. I'll remove it, but maybe in some documentation somewhere it should be mentioned that you will probably need to start your auth url with a leading slash if you are running a single journal instance?

Reply:
Or, more specifically, that the path entered will be treated as a relative path unless it starts with a leading slash?

Reply:
We didn't end up clarifying the documentation here, to my reading.

Reply:
I've pushed a change for this in ff1ce2f6854b8d93f5a06256bd712d26a3ff53e5 - explaining that the WAYF setting can be either a URL or a path, with examples.  I agree with your comments @ctgraham, it's up to admin to enter the value that works for them, be it absolute or relative and shouldn't make any assumptions on what they need.